src/vm/moar/runner/main.c

```diff
@@ -148,7 +148,9 @@ int wmain(int argc, wchar_t *wargv[])
 
     char   *exec_path;
     size_t  exec_path_size;
+#ifndef STATIC_EXEC_PATH
```

Comment:
You could just move the int res down into line 197

Reply:
Done in fd59414.